integration_tests/mockApis/sendLegalMail/oneTimeCode.ts

```diff
@@ -11,7 +11,7 @@ const stubRequestOneTimeCode = (): SuperAgentRequest =>
     request: {
       method: 'POST',
       urlPattern: '/send-legal-mail/oneTimeCode/email',
-      bodyPatterns: [{ matchesJsonPath: '$[?(@.email =~ /.*/i)]' }],
+      bodyPatterns: [{ matchesJsonPath: '$[?(@.email =~ /.*/i)]' }, { matchesJsonPath: '$[?(@.sessionID =~ /.*/i)]' }],
```

Comment:
I forgot to add the `sessionID` request body field when I added these mocks in a previous PR